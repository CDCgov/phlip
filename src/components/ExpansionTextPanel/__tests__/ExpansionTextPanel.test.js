import React from 'react'
import { shallow } from 'enzyme'
import { ExpansionTextPanel } from '../index'

const props = {
  text: 'This is the text of the panel',
  textProps: {
    tooltipText: 'tooltip text'
  },
  dropdownIconProps: {}
}

describe('ExpansionTextPanel', () => {
  test('should render correctly', () => {
    expect(shallow(<ExpansionTextPanel {...props} />)).toMatchSnapshot()
  })
})
