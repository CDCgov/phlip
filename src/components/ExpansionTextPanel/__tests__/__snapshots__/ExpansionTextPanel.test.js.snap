// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpansionTextPanel should render correctly 1`] = `
<Row
  component={
    <WithStyles(Grid)
      item={true}
    />
  }
  displayFlex={true}
  flex={true}
  reverse={false}
  style={
    Object {
      "alignItems": "center",
      "overflow": "hidden",
    }
  }
>
  <WithStyles(Typography)
    noWrap={true}
    style={
      Object {
        "color": "#b2b4b4",
        "flex": 1,
        "minWidth": 0,
      }
    }
    tooltipText="tooltip text"
  >
    This is the text of the panel
  </WithStyles(Typography)>
  <Manager>
    <Reference
      innerRef={[Function]}
    >
      <Component />
    </Reference>
    <Popper
      eventsEnabled={false}
      placement="top-end"
    >
      <Component />
    </Popper>
  </Manager>
</Row>
`;
